spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,32 +4,61 @@
 context 'when switching phases' do
   subject { Season.current }
 
-  describe '#fake_application_phase' do
-    it 'timeshifts the application phase to today' do
-      Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
-        expect(subject).to be_application_period
+  describe 'destined' do
+
+    it 'sends the params to a phase setting method' do
+      phase = 'fake_application_phase'
+      Season::PhaseSwitcher.destined(phase)
+      expect(subject).to_not be_nil
+    end
+
+    context 'when directed to the #fake_application_phase' do
+      it 'sets date range so that the season is open for application submissions' do
+        phase = 'fake_application_phase'
+        Timecop.travel(Season.current.applications_close_at - 1.month) do
+          Season::PhaseSwitcher.destined(phase)
+          expect(subject).to_not be_nil
+          expect(subject).to be_application_period
+        end
       end
     end
-  end
 
-  describe '#fake_coding_phase' do
-    it 'timeshifts the coding phase to today' do
-      Timecop.travel(Season.current.ends_at - 1.week) do
-        Season::PhaseSwitcher.fake_coding_phase
-        expect(subject).to be_started
+    context 'when directed to the #fake_coding_phase' do
+      it 'sets dates so that the summer of code is currently happening' do
+        phase = 'fake_coding_phase'
+        Timecop.travel(Season.current.ends_at - 1.week) do
```

Comment:
Ok. So I'm having a look at the respective files locally.

First of all, in order to be able to use `described_class`, the spec needs to be setup as a proper (model) spec. So start the file with:
```ruby
require 'spec_helper'

describe Season::PhaseSwitcher do
  context 'when switching phases' do
    # ...
```

Once you've done this, the expectation needs to include the `Season` module as well:
```ruby
it "is available as described_class" do
  expect(described_class).to eq(Season::PhaseSwitcher)
end
```
This is necessary because you're out of the `Season` "namespace" in the spec file. You could of course go into it by setting this up like this:
```ruby
require 'spec_helper'

module Season
  describe PhaseSwitcher do
# ...
```
However, I **strongly** advise not to do it! `Season` is not a module but a class and the `PhaseSwitcher` class not set up to be within the `Season` namespace. If it would, it would look like this:
```ruby
module Season
  class PhaseSwitcher
```
However, with Rails 💩 autoloading stuff, you'll run into troubles with this anyways 😉 

Once you got the spec files, namespaces, etc. sorted out, let me know how I can help with the tests themselves.